index.js

```diff
@@ -1,10 +1,6 @@
 module.exports = {
-  extends: [
-    'stylelint-config-recommended',
```

Comment:
**question:** Why are the `stylelint-config-recommended` and `stylelint-config-recommended`scss` packages removed? 🤔

Reply:
I thought this was already included in the new configuration by following the documentation.

Reply:
I did not see any mention of this packages in the [migration guide](https://github.com/stylelint/stylelint/blob/14.0.0/docs/migration-guide/to-14.md) I found, or maybe I misread something?

Reply:
Precisely! I will put them in the configuration again.